dotnet/src/AutoGen.Anthropic/AnthropicClient.cs

```diff
@@ -24,12 +24,24 @@ public sealed class AnthropicClient : IDisposable
     private static readonly JsonSerializerOptions JsonSerializerOptions = new()
     {
         DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
-        Converters = { new ContentBaseConverter(), new JsonPropertyNameEnumConverter<ToolChoiceType>() }
+        Converters =
```

Comment:
Why we need to seperate serialize option and deserialize option here

Reply:
Removed the duplicate one